### PR TITLE
doc: add minutes for meeting 26 Jan 2022

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2022-01-26.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2022-01-26
+
+## Links
+
+* **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>
```
Comment (tniessen) [inline meetings/2022-01-26.md:5]:
```suggestion
* **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>
```

Comment (tniessen) [inline meetings/2022-01-26.md:5]:
```suggestion
* **Recording**: <http://www.youtube.com/watch?v=zL2K-9K8QUw>
```

File: meetings/2022-01-26.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Managing feature requests [#41113](https://github.com/nodejs/node/issues/41113)
+  * Related PR - <https://github.com/nodejs/node/pull/41420>
+  * ask is to please engage in the GitHub repo.
+
+* docs: Clarification around real world risks and use cases of VM module [#40718](https://github.com/nodejs/node/issues/40718)
+  * nothing new to discuss, agreed to leave on the agenda as its important enough that
```
Comment (tniessen) [inline meetings/2022-01-26.md:47]:
```suggestion
  * nothing new to discuss, agreed to leave on the agenda as it is important enough that
```

File: meetings/2022-01-26.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      calendar and group
+    * Added everybody who responded to the issue to the group, invites should be
+      on its way.
+
+* vote: future of primordials [#1104](https://github.com/nodejs/TSC/issues/1104)
+  * Antoine: opened draft PR <https://github.com/nodejs/TSC/pull/1158>
+    * Vote started, deadline is 9 Feb.
+    * Rich 2 questions
+      * Standardize on this tool
+      * Vote, but not consider that a president
```
Comment (tniessen) [inline meetings/2022-01-26.md:67]:
```suggestion
      * Vote, but not consider that a precedent
```

(I think?)

File: meetings/2022-01-26.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+* Handling unintended breaking changes [1156](https://github.com/nodejs/TSC/issues/1156)
+  * Mary opened this because we need to agree on a process so that we can act when it
+    does happen.
+  * Richard, do remember this happening a few times. Approach has been to weigh how
+    quickly we have caught the break. If short time, then point release to revert. Tricker if it’s
+    been a longer period of time.
+  * Danielle, we should come up scenarios, possible and those we have seen and document
+     Them. Include communication steps, etc.
+  * General agreement its a good idea
```
Comment (tniessen) [inline meetings/2022-01-26.md:77]:
```suggestion
  * General agreement it is a good idea
```

**Review by Trott — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]

**Review by tniessen — APPROVED** [review]

**Pull request merged**